src/content/unsupported-browser/UnsupportedBrowser.scss

```diff
@@ -0,0 +1,45 @@
+@import 'src/styles/common';
+
+html,
+body,
+[class='ens-app'] {
+  min-width: initial !important; // !important seems to be the only way to override the min-width set in styles/main.scss
+}
+
+.container {
+  height: 100%;
+  display: flex;
```

Comment:
same here, `display: flex` wont be supported in old browser (IE10 and below) and the styling will be broken.